tests/integration/test_agentd/test_agentd_multi_server/conftest.py

```diff
@@ -4,14 +4,15 @@
 
 import pytest
 
+from wazuh_testing.constants.ports import DEFAULT_SSL_REMOTE_CONNECTION_PORT
 from wazuh_testing.tools.simulators.remoted_simulator import RemotedSimulator
 
 
 @pytest.fixture()
 def start_remoted_simulators(test_metadata) -> None:
     # Servers paremeters
     remoted_server_address = "127.0.0.1"
-    remoted_server_ports = [1514,1516,1517]
+    remoted_server_ports = [DEFAULT_SSL_REMOTE_CONNECTION_PORT,1516,1517]
```

Comment:
port 1516 could be replaced with the constant DEFAULT_SSL_CLUSTER_PORT defined here: src/wazuh_testing/constants/ports.py?

Reply:
These are remoted simulators, that's why I didn't use the cluster constant.